Program.cs

```diff
@@ -13,8 +14,22 @@ public static async Task Main()
                 // Get credentials
                 var config = new ServiceConfig(".env");
 
-                // Create the client
-                var client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            // Create the client
+            IRepositoryApiClient client;
+
+            if (config.AuthorizationType.Equals("LfdsUsernamePassword", StringComparison.OrdinalIgnoreCase))
```

Comment:
very minor, but could you reorder this if check so that `AccessKey` is first? That's the option most are going to use right now, so better for it to be the first thing people see in the sample